cerberus-api-tests/src/main/groovy/com/nike/cerberus/api/CerberusCompositeApiActions.groovy

```diff
@@ -33,10 +33,23 @@ class CerberusCompositeApiActions {
         // Read that the node was updated
         def resp2 = readSecretNode(path, cerberusAuthToken)
         assertEquals(value2, resp2?.'data'?.'value')
+
+        // Since the versions are sorted by timestamp, sleep is used to ensure correct order
```

Comment:
I would add a little more here. This is because these automated test execute very quickly, causing the 'update' and 'delete' timestamp to be the same which causes an issue with sorting

Reply:
I will add that